packages/core/src/electron-browser/window/electron-window-service.ts

```diff
@@ -30,6 +32,18 @@ export class ElectronWindowService extends DefaultWindowService {
         return undefined;
     }
 
+    registerUnloadListener(): void {
+        remote.getCurrentWindow().on('closed', () => this.onUnloadEmitter.fire());
+    }
+
+    /**
+     * Unlike the default implementation, this even is fires when the `ElectronWindow`
```

Comment:
Typo: `event is fired`

On the other hand, people will most likely interact with the `WindowService` interface. Depending on who this comment is intended:
- Theia devs: move the comment to be the class's jsdoc, no need to duplicate this field just to add documentation.
- Theia users: move the comment to `WindowService.onUnload`'s jsdoc mentioning that this behavior occurs on Electron.